gamestonk_terminal/options/op_controller.py

```diff
@@ -148,6 +151,60 @@ def call_calc(self, other_args: List[str]):
         """Process calc command"""
         calculator_model.pnl_calculator(other_args)
 
+    def call_act(self, other_args: List[str]):
+        """Process act command"""
+
+        parser = argparse.ArgumentParser(
+            prog="act",
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            description="This command gets unusual options from fdscanner.com",
+        )
+        parser.add_argument(
+            "-n" "--num",
+            dest="num",
+            type=int,
+            default=20,
+            help="Number of options to show.  Each scraped page gives 20 results.",
+        )
+
+        parser.add_argument(
+            "--sortby",
+            dest="sortby",
+            default="Vol/OI",
+            choices=["Option", "Vol/OI", "Vol", "OI", "Bid", "Ask"],
+            help="Column to sort by.  Vol/OI is the default and typical variable to be considered unusual.",
+        )
+        parser.add_argument(
+            "-a",
+            "--ascending",
+            dest="ascend",
+            default=False,
+            action="store_true",
+            help="Flag to sort in ascending order",
+        )
+
+        parser.add_argument(
+            "--export",
+            choices=["csv", "json", "xlsx"],
+            default="",
+            dest="export",
+            help="Export dataframe data to csv,json,xlsx file",
+        )
+
+        try:
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
+            num = ns_parser.num
+            sort_column = ns_parser.sortby
+            export = ns_parser.export
+            ascending = ns_parser.ascend
+            fdscanner_view.display_options(num, sort_column, export, ascending)
```

Comment:
I think here I mislead you, I meant doing something like:

```
fdscanner_view.display_options(num=ns_parser.num, 
             sort_column=ns_parser.sortby, 
             export=ns_parser.export, 
            ascending=ns_parser.ascend)
```
and not declaring new variables

Reply:
I see